slippi-stats.js

```diff
@@ -91,33 +112,56 @@ var final_player_name = user_player
 
```

Comment:
```suggestion
let real_player_code
let final_opponent_name
let real_opponent_code
```
Some searches fail cause of uninitialized variables at the start like real_player_code, final_opponent_name, real_opponent_code (like you fixed in 3fd6e39)